build/common.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <!-- Extensions can have independent versions and only increment when released -->
     <Version>3.0.0$(VersionSuffix)</Version>
-    <ExtensionsVersion>3.0.5$(VersionSuffix)</ExtensionsVersion> <!-- WebJobs.Extensions -->
+    <ExtensionsVersion>3.0.6$(VersionSuffix)</ExtensionsVersion> <!-- WebJobs.Extensions -->
```

Comment:
In addition, to generate artifacts without the prerelease version suffix, you'll need to tag the commit, e.g. "extensions-v3.0.6". See the commit history for other examples.